### ISSUE TITLE
streamable.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#1879]
### Basic Information

**Test link:** `https://www.streamable.com`
**Category:** breakage
**Reported from:** Finland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.126

### Additional Message

Original URL: `https://www.unknowncheats.me/forum/counterstrike-global-offensive/354646-choke-packets.html`

If the ads are blocked on a embedded video, the video simply refuses to play at all.

**Comment by llacb47** [comment]
Ads on the Original URL:
```
unknowncheats.me##.alt1 > center:has-text(sponsored)
unknowncheats.me###hmmk
```

For the streamable embed:
Test embed here `https://misterdocafe.blogspot.com/2019/09/o-rei-vai-nu.html`

Add 
```
! https://forums.lanik.us/viewtopic.php?f=64&t=40868
||embed-cdn.com^
```

**Comment by jspenguin2017** [comment]
@okiehsch

**Comment by okiehsch** [comment]
`https://misterdocafe.blogspot.com/2019/09/o-rei-vai-nu.html`
the video plays without issue on my end.

**Comment by llacb47** [comment]
The embed waits for the ad to be served. If not, it forces play after a delay.

**Comment by jspenguin2017** [comment]
@okiehsch For `https://www.unknowncheats.me/forum/counterstrike-global-offensive/354646-choke-packets.html`: 
![image](https://user-images.githubusercontent.com/7283682/65809659-d814e680-e15c-11e9-9992-ffe6fd97b0f9.png)

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://streamable.com/zt2lx`
**Category:** ads
**Reported from:** Canada
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.78 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.135

### Additional Message

**Comment by jspenguin2017** [comment]
> @llacb47 
> Test embed here `https://misterdocafe.blogspot.com/2019/09/o-rei-vai-nu.html`

Works fine for me. 

---

> https://www.unknowncheats.me/forum/counterstrike-global-offensive/354646-choke-packets.html

Splitting off to another issue.
